manifests/init.pp

```diff
@@ -45,186 +51,217 @@
 #   each subarray matches a member IP addresses.
-#   Defaults to ['localhost']
+#   Default: ['localhost']
 #
 # [*quorum_members_ids*]
 #   Array of quorum member IDs. Persistent IDs are required for the dynamic
 #   config of a corosync cluster and when_set_votequorum is set to true.
 #   Should be used only with the quorum_members parameter.
-#   Defaults to undef
+#   Default: undef
 #
 # [*quorum_members_names*]
 #   Array of quorum member names. Persistent names are required when you
 #   define IP addresses in quorum_members.
-#   Defaults to undef
+#   Default: undef
 #
 # [*token*]
 #   Time (in ms) to wait for a token
+#   Default: undef
 #
 # [*token_retransmits_before_loss_const*]
-#   How many token retransmits before forming a new configuration
+#   How many token retransmits before forming a new configuration.
+#   Default: undef
+#
+# [*compatibility*]
+#   Default: undef
 #
 # [*enable_corosync_service*]
 #   Whether the module should enable the corosync service.
+#   Default: true
 #
 # [*enable_pacemaker_service*]
 #   Whether the module should enable the pacemaker service.
+#   Default: true
 #
 # [*manage_pacemaker_service*]
 #   Whether the module should try to manage the pacemaker service in
 #   addition to the corosync service.
-#   Defaults to true on RedHat based operating systems version 7 or greater.
-#   Defaults to true on Ubuntu version 14.04 or greater.
-#   Defaults to false on all other operating systems.
+#   Default (Red Hat based >= 7): true
+#   Default (Ubuntu >= 14.04):    true
+#   Default (otherwise):          false
 #
 # [*enable_pcsd_service*]
 #   Whether the module should enable the pcsd service.
+#   Default: true
 #
 # [*manage_pcsd_service*]
 #   Whether the module should try to manage the pcsd service in addition to the
 #   corosync service.
 #   pcsd service is the GUI and the remote configuration interface.
-#   Defaults to false
+#   Default: false
 #
 # [*cluster_name*]
 #   This specifies the name of cluster and it's used for automatic
 #   generating of multicast address.
+#   Default: undef
 #
 # [*join*]
 #   This timeout specifies in milliseconds how long to wait for join messages
 #   in the membership protocol.
-#   Default to 50
+#   Default: undef
 #
 # [*consensus*]
 #   This timeout specifies in milliseconds how long to wait for consensus to be
 #   achieved before starting a new round of membership configuration.
 #   The minimum value for consensus must be 1.2 * token. This value will be
 #   automatically calculated at 1.2 * token if the user doesn't specify a
 #   consensus value.
-#   Defaults to false,
+#   Default: undef
 #
 # [*max_messages*]
 #   This constant specifies the maximum number of messages that may be sent by
 #   one processor on receipt of the token. The max_messages parameter is limited
 #   to 256000 / netmtu to prevent overflow of the kernel transmit buffers.
-#   Defaults to 17
+#   Default: undef
```

Comment:
I wouldn't focus too much on these, since we should just move it to Puppet strings, which will auto-document the defaults and types anyway.